File: Coding/coding modul 1/PRAK104-2410817110007-MUHAMMAD-HAMKA-AKBARI.c
#include <stdio.h>

int main()
{
    double sepatu_A = 400000;
    double sepatu_B = 350000;

    printf("Harga sepatu A adalah %.0lf\n", sepatu_A);
    printf("\nHarga sepatu B adalah %.0lf\n", sepatu_B);

    double diskon1 = 0.13;
    double diskon2 = 0.21;

    double harga_diskon1 = sepatu_A - (diskon1 * sepatu_A);
    double harga_diskon2 = sepatu_B - (diskon2 * sepatu_B);

    printf("\nSepatu A mendapat diskon 13%% sehingga harganya menjadi %.0lf\n", harga_diskon1);
    printf("\nSepatu A mendapat diskon 21%% sehingga harganya menjadi %.0lf", harga_diskon2);

    return 0;
}

File: coding modul 1/PRAK104-2410817110007-MUHAMMAD-HAMKA-AKBARI.py
sepatu_A = 400000
sepatu_B = 350000

print(f"Harga sepatu A adalah {sepatu_A}")
print(f"\nHarga sepatu B adalah {sepatu_B}")

diskon1 = 0.13 
diskon2 = 0.21 

harga_diskon1 = sepatu_A - (sepatu_A * diskon1)
harga_diskon2 = sepatu_B - (sepatu_B * diskon2)

print(f"\nSepatu A mendapat diskon 13% sehingga harganya menjadi {harga_diskon1:.0f}")
print(f"\nSepatu A mendapat diskon 21% sehingga harganya menjadi {harga_diskon2:.0f}")